newIDE/app/src/EventsSheet/EventsTree/index.js

```diff
@@ -84,6 +89,12 @@ class EventContainer extends Component<EventsContainerProps, {||}> {
     this.props.onEventContextMenu(domEvent.clientX, domEvent.clientY);
   };
 
+  renderObjectThumbnail = (objectName) =>{
+    const { project, layout } = this.props;
+    const object = layout.getObject(objectName)
```

Comment:
This will crash if the object does not exist, and will crash if the object is global

Reply:
didnt we have a method somewhere that does all the checks already?

Reply:
We have one that you added that is used for instances in another PR, but not sure if you can reuse exactly the same? If needed, make a slightly more generic. I think the other PR will be merged anyway before this one.

Reply:
If I make it slightly more generic like this:
```
  getObject(objectName, layout, project) {
    var associatedObject = null;
    if (layout.hasObjectNamed(objectName))
      associatedObject = layout.getObject(objectName);
    else if (project.hasObjectNamed(objectName))
      associatedObject = project.getObject(objectName);
    return associatedObject;
  }
```
Can we reuse it in this case as well? I will amend it in my other pull if so